docs/modules/databend.md

```diff
@@ -0,0 +1,75 @@
+# Databend
+
+Not available until the next release of testcontainers-go <a href="https://github.com/testcontainers/testcontainers-go"><span class="tc-version">:material-tag: main</span></a>
+
+## Introduction
+
+The Testcontainers module for Databend.
+
+## Adding this module to your project dependencies
+
+Please run the following command to add the Databend module to your Go dependencies:
+
+```
+go get github.com/testcontainers/testcontainers-go/modules/databend
+```
+
+## Usage example
+
+<!--codeinclude-->
+[Creating a Databend container](../../modules/databend/examples_test.go) inside_block:runDatabendContainer
+<!--/codeinclude-->
+
+## Module Reference
+
+### Run function
+
+- Not available until the next release of testcontainers-go <a href="https://github.com/testcontainers/testcontainers-go"><span class="tc-version">:material-tag: main</span></a>
+
+!!!info
+    The `RunContainer(ctx, opts...)` function is deprecated and will be removed in the next major release of _Testcontainers for Go_.
```

Comment:
bug: There is no such method, might be an old template?

Reply:
Yes, it is generated by the template.

Reply:
Have removed.